packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -99,8 +110,13 @@ export function Choices({
         const choice = task.choices?.[choiceId] || {}
         const selected = selectedChoiceIds.indexOf(choiceId) > -1
         const src = task.images?.[choice.images?.[0]] || ''
-        const hasFocus = autoFocus && (index === focusIndex)
-        const tabIndex = (index === focusIndex) ? 0 : -1
+        const hasFocus = index === focusIndex
+        let tabIndex = -1
+        if (focusIndex === null && index === 0) {
+          tabIndex = 0
+        } else if (index === focusIndex) {
+          tabIndex = 0
+        }
 
         return (
           <ChoiceButton
```

Comment:
Now that `StyledGrid` is an unordered list, the individual buttons should be wrapped in `li`. Best to keep the `ul` and `li` elements together in the code, so I would put the list items here.